app/SearchApi/SearchApi.Web/Search/MatchFoundConsumer.cs

```diff
@@ -10,16 +13,21 @@ public class MatchFoundConsumer : IConsumer<MatchFound>
 
         private readonly ILogger<MatchFoundConsumer> _logger;
 
-        public MatchFoundConsumer(ILogger<MatchFoundConsumer> logger)
+        private readonly ISearchApiNotifier _serSearchApiNotifier;
+
+        public MatchFoundConsumer(ISearchApiNotifier serSearchApiNotifier, ILogger<MatchFoundConsumer> logger)
         {
-            this._logger = logger;
+            _serSearchApiNotifier = serSearchApiNotifier;
```

Comment:
are u sure serSearchApiNotifier is the variable name you want?

Reply:
thanks ;-)